.github/workflows/build-container.yml

```diff
@@ -0,0 +1,67 @@
+name: Release ingress-netperf container image
+
+on:
+  pull_request:
+    paths:
+      - 'containers/Containerfile' # Trigger only when Dockerfile changes in a pull request
```

Comment:
Let's track the full containers dir rather than only one file

Reply:
Sure, new changes coming along.